blazor/file-manager/flat-data.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: Flat data rendering in Blazor FileManager component | Syncfusion
+description: Checkout and learn here all about flat data rendering in Syncfusion Blazor FileManager component and more.
+platform: Blazor
+control: File Manager
+documentation: ug
+---
+
+# Flat data rendering in Blazor FileManager component
+
+This section briefly explains how to render the FileManager component using a flat data object. With the latest enhancement, the File Manager component now employs a flat data object for rendering, eliminating the need to define `FileManagerAjaxSettings`. Instead, in the documentation code, a physical service is injected for file actions, and the corresponding file action's response can be retrieved from the locally injected service with the help of new event support for actions performed with the FileManager component.
```

Comment:
In the documentation code, a physical service is injected to handle file actions, and the resulting file action response can be obtained locally through the newly added event support for actions performed with the FileManager component.

Reply:
@keerthanaRajendran as per the suggestion, modified the documentation content.
https://github.com/syncfusion-content/blazor-docs/pull/4076/commits/aba37ec5c56870af59b35a357c7d63f552140496